CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Modal in admin/cms/store pages
```

Comment:
```suggestion
- Favicon modal in admin/cms/store pages
```

Reply:
This is for every component using modal in this case. 

The favicon is only in this example